dp-terraform/helm/watches.yaml

```
@@ -0,0 +1,4 @@
+- group: fleet.stackrox.io
```

Is this following any earlier decision that I'm unaware of?
If not, how about
```suggestion
- group: cloud.stackrox.io
```

---

There was no earlier decision, I've changed the group as per your suggestion.